src/modules/create-wallet/components/CreateWalletKeystore.vue

```diff
@@ -280,6 +282,7 @@ export default {
     },
     downloadWallet() {
       this.$refs.downloadLink.click();
+      this.trackCreateWallet('keystore');
```

Comment:
can we import wallettypes and use that, instead of hardcoding `keystore`

Reply:
done!